templates/common/git/git_test.go

```diff
@@ -63,55 +63,54 @@ func TestClone(t *testing.T) {
 	t.Parallel()
 
 	cases := []struct {
-		name        string
-		remote      string
-		branchOrTag string
-		wantErr     string
+		name    string
+		remote  string
+		version string
+		wantErr string
```

Comment:
I think it's worth to add a test case when the sha is not valid, this is to add coverage for the regexp. Please correct me if it's tested elsewhere.